index.js

```diff
@@ -2,7 +2,12 @@
 var Promise = require('native-or-bluebird')
 
 module.exports = function (interval) {
-  return new Promise(function (resolve) {
-    setTimeout(resolve, interval || 0)
+  var timeoutId
```

Comment:
Okay, I was just following MDN
https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setTimeout

Will change

Typed using my thumbs..

> On Jun 9, 2016, at 6:55 AM, haoxin <notifications@github.com> wrote:
> 
> In index.js:
> 
> > @@ -2,7 +2,12 @@
> >  var Promise = require('native-or-bluebird')
> >  
> >  module.exports = function (interval) {
> > -  return new Promise(function (resolve) {
> > -    setTimeout(resolve, interval || 0)
> > +  var timeoutId
> prefer timer, in node, setTimeout return a obj!
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>